sigpac_tools/_globals.py

```diff
@@ -0,0 +1,23 @@
+BASE_URL = "https://sigpac.mapama.gob.es"
+
+PROVINCES_BY_COMMUNITY = {
```

Comment:
Please add a comment explaining what is this what the number means so that it can be updated, if it may ever change (What happens if a community leaves Spain for example, which number do I need to delete)